c-dependencies/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -1141,7 +1141,25 @@ bool Request::set_cache_override(JSContext *cx, JS::HandleObject self,
                       JS::ObjectValue(*override));
   return true;
 }
+bool Request::apply_auto_decompress(JSContext *cx, JS::HandleObject self) {
+  MOZ_ASSERT(cx);
+  MOZ_ASSERT(is_instance(self));
+  auto decompress =
+      JS::GetReservedSlot(self, static_cast<uint32_t>(Request::Slots::AutoDecompress)).toBoolean();
+  if (!decompress) {
+    return true;
+  }
 
+  fastly_error_t err;
+  fastly_content_encodings_t encodings_to_decompress = 0;
+  encodings_to_decompress |= FASTLY_CONTENT_ENCODINGS_GZIP;
```

Comment:
Why not also include Brotli?

Reply:
It isn't supported by the host, the host only supports gzip (for now)